temporian/io/test/numpy_test.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+from absl.testing import absltest
+
+from temporian.implementation.numpy.data.io import event_set
+from temporian.io.numpy import to_numpy
+
+
+class NumpyTest(absltest.TestCase):
+
+    def test_correct(self):
+        evset = event_set(
+            timestamps=['2023-11-08T17:14:38', '2023-11-29T21:44:46'],
+            features={
+                "feature_1": [0.5, 0.6],
+                "my_index": ["red", "blue"],
+            },
+            indexes=["my_index"],
+        )
+
+        result = to_numpy(evset)
+
+        expected_timestamps = np.array(['2023-11-08T17:14:38', '2023-11-29T21:44:46'],
+                                       dtype='datetime64[s]')
+        expected_feature_1 = np.array([0.5, 0.6])
+        expected_my_index = np.array(["red", "blue"])
```

Comment:
Nit: Could we define this as a dictionary?

something like:
``` python
expected = {
    "timestamps": np.array(['2023-11-08T17:14:38', '2023-11-29T21:44:46'],
                                dtype='datetime64[s]'),
    "feature_1": np.array([0.5, 0.6]),
    "my_index": np.array(["red", "blue"]),
}
```

It would make the code a bit more compact and allow the asserts below to be on a for-loop

Reply:
modified to dict and added a loop to check each key